lib/gds_api/publishing_api_v2.rb

```diff
@@ -1,6 +1,14 @@
 require_relative 'base'
 
 class GdsApi::PublishingApiV2 < GdsApi::Base
+  # Put a content item
+  #
+  # This creates a new draft item, which will be sent to the draft content store.
+  #
+  # @param content_id [UUID]
+  # @param payload [Hash] A valid content item
```

Comment:
`A valid x` assumes I already know a lot about how the publishing platform works. Can we refer people to the content schemas and/or some examples?